execute/exectypes/commit_data.go

```diff
@@ -41,6 +41,16 @@ type CommitData struct {
 	MessageTokenData []MessageTokenData `json:"messageTokenData"`
 }
 
+func CompareCommitData(i, j CommitData) bool {
```

Comment:
Doc comment explaining the significance of the implementation would be good so that we don't modify this erroneously in the future.